src/main/java/com/github/maxopoly/MemeMana/command/CmdMana.java

```diff
@@ -0,0 +1,91 @@
+package com.github.maxopoly.MemeMana.command;
+
+import vg.civcraft.mc.civmodcore.command.PlayerCommand;
+import vg.civcraft.mc.namelayer.NameAPI;
+import com.devotedmc.ExilePearl.ExilePearlPlugin;
+import com.devotedmc.ExilePearl.ExilePearl;
+import com.github.maxopoly.MemeMana.model.MemeManaPouch;
+import com.github.maxopoly.MemeMana.model.MemeManaOwner;
+import com.github.maxopoly.MemeMana.MemeManaIdentity;
+import com.github.maxopoly.MemeMana.MemeManaPlugin;
+import org.bukkit.entity.Player;
+import org.bukkit.command.CommandSender;
+import java.util.UUID;
+import java.util.List;
+import java.util.LinkedList;
+
+public class CmdMana extends PlayerCommand {
+	public CmdMana(String name) {
+		super(name);
+		setIdentifier("mana");
+		setDescription("The base command for Meme Mana");
+		setUsage("\n/mana show -- Show your current mana\n/mana refill [Amount] -- Refill the pearl in your hand by an amount\n/mana transfer <Player> <Amount> -- Transfer some mana to another player");
+		setArguments(1,3);
+	}
+
+	public boolean execute(CommandSender sender, String [] args) {
+		if (args[0] == "show") {
+			showManaAmount(sender);
+		} else if (args[0] == "refill") {
+			doRefill(sender,args);
+		} else if (args[0] == "transfer") {
+			doTransfer(sender,args);
+		}
+		return true;
+	}
+
+	public void showManaAmount(CommandSender sender) {
+		if (!(sender instanceof Player)) {
+			msg("Can't show your own mana from console. Try /mana inspect instead");
```

Comment:
Done use that method, I dont know why it even exists. It's very much an antipattern and I think it's older than spigots ChatColor API, which allows you to do colors in a much cleaner and version consistent way.

Reply:
You're talking about `msg`? That's what ExilePearl uses at least, idk anything about Spigot chat colors...

Reply:
Yeah, msg is a relic and shouldnt be used. ChatColor is an enum that's part of Spigots API, you can just concatenate it with strings when sending messages to players. Example:

https://github.com/DevotedMC/FactoryMod/blob/master/src/main/java/com/github/igotyou/FactoryMod/interactionManager/FurnCraftChestInteractionManager.java#L73